presto-product-tests/libs/download.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -e;
+
+HIVE_JDBC_DOWNLOAD_URL='http://bdch-ftp.td.teradata.com:8081/nexus/service/local/artifact/maven/redirect?r=test-framework&g=org.apache.hive&a=hive-jdbc-all&v=1.0-SNAPSHOT&e=jar'
```

Comment:
I removed that, now it requires https://github.com/prestodb/presto-hive-jdbc/pull/1 to be accepted and merged. Then presto-hive-jdbc needs to be deployed on maven repositories.